deployment/roles/reverseproxy/templates/security-headers.toml.j2

```diff
@@ -0,0 +1,35 @@
+[tls.options]
+    [tls.options.default]
+      minVersion = "VersionTLS12"
+      sniStrict = true
+      cipherSuites = [
+          "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+          "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+          "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
+          "TLS_AES_128_GCM_SHA256",
+          "TLS_AES_256_GCM_SHA384",
+          "TLS_CHACHA20_POLY1305_SHA256",
+          "TLS_FALLBACK_SCSV"
+      ]
+
+[http.middlewares]
+    [http.middlewares.security-headers.headers]
+      # CORS
+      AccessControlAllowMethods = ["GET", "OPTIONS", "PUT"]
+      AccessControlAllowOrigin = "origin-list-or-null"
+      AccessControlMaxAge = 100
+      #AddVaryHeader = true
```

Comment:
Dead code? Do we need it or not? If not, you can remove the line.

Reply:
wops, left it in there by accident while testing.